sources/T-Editing-Guidelines-201602.adoc

```diff
@@ -0,0 +1,843 @@
+
+[[a_E_3_1Deletion_od_a_clause_table_or_figure]]
+==== Deletion of a clause, table or figure
+
+The subsequent clauses, tables or figures should not be renumbered, and the deleted number should not be reused.
+
+[[a_E_3_2Insertion_of_a_clause_table_or_figure]]
+==== Insertion of a clause, table or figure
+
+Previous clause, figure or table numbers will be extended with letters in order to designate the inserted clauses, tables or figures.
+
+_Example:_ If several clauses are to be inserted, for instance, between clauses 6.1 and 6.2, they should be numbered as 6.1a, 6.1b, etc.
+
+[[a_E_3_3Renumbering]]
+==== Renumbering
+
+When issuing a completely revised edition, clauses, figures and tables will be renumbered sequentially.
+
+
+[appendix,obligation=informative]
+== Appendices
+
+Appendices do not form an integral part of a Recommendation. It should be noted that while a bibliography is considered an appendix, and thus is non-normative, it is not explicitly labelled as an appendix. The text "(This appendix does not form an integral part of this Recommendation.)" does not follow the "Bibliography" heading.
+
+The sole reference to a normative reference listed in clause 2 should not be made from an appendix. The sole reference to a non-normative reference such as <<b-ITU-T_K.35>> may appear in an appendix.
+
+
+[bibliography]
+[[a_Bibliography]]
+== Bibliography
+
+* [[[b-ITU-T_A.1,b-ITU-T A.1]]], Recommendation ITU-T A.1 (2012), _Working methods for study groups of the ITU Telecommunication Standardization Sector_.
+
+* [[[b-ITU-T_A.2,b-ITU-T A.2]]], Recommendation ITU-T A.2 (2012), _Presentation of contributions to the ITU Telecommunication Standardization Sector_.
+
+* [[[b-ITU-T_A.4,b-ITU-T A.4]]], Recommendation ITU-T A.4 (2012), _Communication process between the ITU Telecommunication Standardization Sector and forums and consortia_.
+
+* [[[b-ITU-T_A.6,b-ITU-T A.6]]], Recommendation ITU-T A.6 (2012), _Cooperation and exchange of information between the ITU Telecommunication Standardization Sector and national and regional standards development organizations_.
+
+* [[[b-ITU-T_A.8,b-ITU-T A.8]]], Recommendation ITU-T A.8 (2008), _Alternative approval process for new and revised ITU-T Recommendations_.
+
+* [[[b-ITU-T_A.13,b-ITU-T A.13]]], Recommendation ITU-T A.13 (2000), _Supplements to ITU‑T Recommendations_.
+
+* [[[b-ITU-T_G.108.2,b-ITU-T G.108.2]]], Recommendation ITU-T G.108.2 (2007), _Transmission planning aspects of echo cancellers_.
+
+* [[[b-ITU-T_G.709,b-ITU-T G.709]]], Recommendation ITU-T G.709/Y.1331 (2012), _Interfaces for the optical transport network_.
+
+* [[[b-ITU-T_K.35,b-ITU-T K.35]]], Recommendation ITU-T K.35 (1996), _Bonding configurations and earthing at remote electronic sites_.
+
+* [[[b-ITU-T_Rap,b-ITU-T Rap]]], ITU-T, _Manual for Rapporteurs and Editors_, 12 February 2010. +
+http://www.itu.int/oth/T0A0F000006/en[http://www.itu.int/oth/T0A0F000006/en]
```

Comment:
I think we probably don't want manual formatting or line breaks with references. @opoudjis your two cents please.

Reply:
Nermina's brief was to replicate source markup to the extent possible, so that we don't raise questions about what Metanorma can't do. I am normally as vehemently opposed as you are to capricious format; in my opinion though this instance is harmless, and I'd prefer it to stay.

Reply:
@opoudjis Metanorma does things semantically. Non-semantic rendering should be either removed or made semantic.